starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,57 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+.text {
+    font-size: 11px;
+    font-weight: 200;
+    margin-left: 29%;
+    margin-right: 29%;
+    font-family: Arial;
+    margin-bottom: 20px;
+}
+
+h1 {      
+    font-size: 60px;
+    text-align: center;
+    color: white;
+
+}
+
+h2 {
+    font-size: 16px;
+    line-height: 4em;
+    
+}
+
+h3 {
+    color: black;
+    font-size: 14px;
+    text-align: center;
+    padding-top: 50px;
+}
+
+.image img {
+    width: 300px;
+    margin-left: 29%;
+    margin-right: 29%;
+
+}
```

Comment:
here instead of having the margins set to fixed percentages you could display the image as block and have the margin: auto property center the image

.```
image img {
    width: 300px;
  display:block;
margin: 0 auto; // apply equal margin on the left and right
}
```